OutOfSchool/OutOfSchool.BusinessLogic/Models/CompetitiveEvent/CompetitiveEventDto.cs

```diff
@@ -1,14 +1,21 @@
-﻿namespace OutOfSchool.BusinessLogic.Models.CompetitiveEvent;
+﻿using Microsoft.AspNetCore.Mvc;
+using OutOfSchool.BusinessLogic.Util.JsonTools;
+using System.ComponentModel.DataAnnotations;
+
+namespace OutOfSchool.BusinessLogic.Models.CompetitiveEvent;
 
 public class CompetitiveEventDto : CompetitiveEventBaseDto
 {
     public Guid Id { get; set; }
     public bool IsDeleted { get; set; }
-
     public uint Rating { get; set; } = 0;
     public uint NumberOfRatings { get; set; } = 0;
     public string InstitutionHierarchy { get; set; }
     public List<long> DirectionIds { get; set; }
+    [MaxLength(256)]
+    public string CoverImageId { get; set; } = string.Empty;
 
+    [ModelBinder(BinderType = typeof(JsonModelBinder))]
```

Comment:
this is display only model, so don't really need json binder as it is not used for multipart forma data uploads.